test/JDBC/expected/BABEL-4217-vu-prepare.out

```diff
@@ -0,0 +1,292 @@
+VALUES ('John'),
+       ('Jane');
+GO
+~~ROW COUNT: 2~~
+
+
+-- Used UNIQUE & Created Table first and then use alter table(in BABEL-4217-vu-verify.sql) and placed not null before UNIQUE Constraint
+CREATE TABLE BABEL_4217_vu_prepare_t12 (
+    FIRSTNAME VARCHAR(50)
+);
+GO
+
+INSERT INTO BABEL_4217_vu_prepare_t12 (FIRSTNAME)
+VALUES ('John'),
+       ('Jane');
+GO
+~~ROW COUNT: 2~~
+
+
+-- Used PRIMARY KEY, placed NOT NULL before Identity Column & added a check constraint before identity column
+CREATE TABLE BABEL_4217_vu_prepare_t13 (
+    ID INT PRIMARY KEY NOT NULL CHECK(ID < 5) IDENTITY(1,1),
+    FIRSTNAME VARCHAR(50)
+);
+GO
+
+INSERT INTO BABEL_4217_vu_prepare_t13 (FIRSTNAME)
+VALUES ('John'),
+       ('Jane');
+GO
+~~ROW COUNT: 2~~
+
+
+-- Used PRIMARY KEY, placed NOT NULL before Identity Column & added default constraint before identity column (it should give error)
+CREATE TABLE BABEL_4217_vu_prepare_t14 (
+    ID INT PRIMARY KEY NOT NULL DEFAULT 0 IDENTITY(1,1),
+    FIRSTNAME VARCHAR(50)
+);
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: both default and identity specified for column "id" of table "babel_4217_vu_prepare_t14")~~
+
+
+-- Used PRIMARY KEY & Created Table first and then use alter table(in BABEL-4217-vu-verify.sql), placed not null before identity column and added "check" constraint before identity column
+CREATE TABLE BABEL_4217_vu_prepare_t15 (
+    FIRSTNAME VARCHAR(50)
+);
+GO
+
+INSERT INTO BABEL_4217_vu_prepare_t15 (FIRSTNAME)
+VALUES ('John'),
+       ('Jane');
+GO
+~~ROW COUNT: 2~~
+
+
+-- Used PRIMARY KEY & Created Table first and then use alter table(in BABEL-4217-vu-verify.sql), placed not null before identity column and added "default" constraint before identity column
+CREATE TABLE BABEL_4217_vu_prepare_t16 (
+    FIRSTNAME VARCHAR(50)
+);
+GO
+
+INSERT INTO BABEL_4217_vu_prepare_t16 (FIRSTNAME)
+VALUES ('John'),
+       ('Jane');
+GO
+~~ROW COUNT: 2~~
+
+
+-- Used PRIMARY KEY, placed NOT NULL before Identity Column & added a check constraint before identity column
+CREATE TABLE BABEL_4217_vu_prepare_t17 (
+    ID INT PRIMARY KEY NOT NULL CHECK(ID > 2) IDENTITY(1,1),
+    FIRSTNAME VARCHAR(50)
+);
+GO
+
+-- It should give error (check constraint violation)
+INSERT INTO BABEL_4217_vu_prepare_t17 (FIRSTNAME)
+VALUES ('John'),
+       ('Jane');
+GO
+~~ERROR (Code: 547)~~
+
+~~ERROR (Message: new row for relation "babel_4217_vu_prepare_t17" violates check constraint "babel_4217_vu_prepare_t17_id_check")~~
+
+
+--INSERT BULK queries are not op query, we are just handling the syntax from antlr parser side.
+-- Used PRIMARY KEY, placed NOT NULL before Identity Column & used INSERT BULK query
+CREATE TABLE BABEL_4217_vu_prepare_t18 (
+    ID INT PRIMARY KEY NOT NULL IDENTITY(1,1),
+    FIRSTNAME VARCHAR(50)
+);
+GO
+
+INSERT INTO BABEL_4217_vu_prepare_t18 (FIRSTNAME)
+VALUES ('John'),
+       ('Jane');
+GO
+~~ROW COUNT: 2~~
+
+
+INSERT BULK BABEL_4217_vu_prepare_t18 (
```

Comment:
All insert bulk queries should be in verify file instead of prepare. Additionally, let's try to minimize the number of tables we are creating for testing.

Reply:
Done